src/util/apis/create-client.ts

```diff
@@ -81,7 +82,7 @@ export function clientCall<T>(action: { (cb: ServiceCallback<any>): void }): Pro
 //
 export interface ClientResponse<T> {
```

Comment:
This interface is used in show.ts in getProperties method. and since response field is not being used anywhere it could be removed. Only result field is actually needed. So, all things considered we probably can remove ms-rest-js completely.